www/src/redux/hooks.ts

```diff
@@ -0,0 +1,7 @@
+import { TypedUseSelectorHook, useDispatch, useSelector } from 'react-redux';
+
+import type { AppDispatch, RootState } from './store';
+
+// Use throughout your app instead of plain `useDispatch` and `useSelector`
+export const useAppDispatch = (): AppDispatch => useDispatch<AppDispatch>();
```

Comment:
This is so dispatch and selector are typed right? Like this is to use `useDispatch` and `useSelector` with TS?

Reply:
Yes, this corresponds to the [TypeScript Quick Start | Redux](https://redux.js.org/tutorials/typescript-quick-start#define-typed-hooks) section